test/buddy/auth/backends/token_tests.clj

```diff
@@ -51,6 +55,13 @@
     {:headers {"authorization" header}}))
 
 (deftest jws-tests
+  (testing "Jws token backend authentication with RSA key"
+    (let [request (make-jws-request jws-data rsa-privkey)
```

Comment:
The `make-jws-request` generates request with `:hs256` algorithm by default. But you are passing it a rsa privkey, it is not expected to work. See this commit that has a correct test case for your use case: https://github.com/funcool/buddy-auth/commit/024ae8d93baf758f14225cfe864a88ba3d23e526